src/index.js

```diff
@@ -27,6 +48,15 @@ const withMarkdoc =
 
         return config;
       },
+      
+      // Add Turbopack configuration
+      turbopack: {
+        ...nextConfig.turbopack,
+        rules: {
+          ...nextConfig.turbopack?.rules,
+          ...turbopackRules,
+        },
+      },
```

Comment:
What about something like this:
```suggestion
      turbopack: nextConfig.turbopack ? {
        ...nextConfig.turbopack,
        rules: {
          ...nextConfig.turbopack.rules,
          ...turbopackRules,
        },
      } : undefined,
```
So that turbopack isn't set accidentally.